Card/src/main/java/com/paypal/android/card/ConfirmedPaymentSource.kt

```diff
@@ -0,0 +1,9 @@
+package com.paypal.android.card
+
+data class ConfirmedPaymentSource(
```

Comment:
what do you think of having something like this:
```
data class OrderDataResponse(
  val orderID: String
  val status: OrderStatus     // I know this is String but we can do enum so it's friendlier to integration merchants
  val intent: Intent?    // Same with String to enum
  val payementSource: PaymentSourceResponse?     // ==> In this case can be CardResponse having last_digits, brands, etc. but can extends to other payment methods in the future
)
```

That way we can use the same response type for everything, so:
- it's clearer to both us and merchants
- no need to implement multiple response types that look kinda similar to each other.

But also downside:
- Can we assume this? maybe good to confirm with API team whether these responses are the same object from the server side? 
- what if our API changes these format in the future?

Reply:
sorry for the long comment, but please lemme know your thoughts :smile:

Reply:
I think this would be good to discuss more as a team - a single response object seems like it would work well now, but feels like it will cause problems later if we have to force all payment method responses to conform to one object type. Also depending on API changes, it could lead to this class/module needing to know details about specific payment methods that exist outside this module(ex: if the response for a Venmo order contains venmo-specific fields, we would have to update this), which we want to avoid. 

Another idea is a having a `BaseOrderDataResponse` class that lives in the core module that could follow the current expected pattern, and a `CardOrderDataResponse` that extends it and contains any card-specific fields

Reply:
Yeah I like the idea of having more simplified types to make things easier to comprehend for the merchant. We have done similar things in BT Android with base types. But overall I agree, keeping things similar across different payment methods will probably lead to increased adoption.